pytorch_lightning/core/hooks.py

```diff
@@ -295,6 +295,23 @@ def on_before_zero_grad(self, optimizer: Optimizer) -> None:
             optimizer: The optimizer for which grads should be zeroed.
         """
 
+    def on_before_backward(self, loss: torch.Tensor) -> None:
+        """
+        Override on_before_backward with your own implementation if you need to.
+
+        Args:
+            loss: Loss is already scaled by accumulated grads and possibly scaled by Mixed Precision.
```

Comment:
```suggestion
            loss: loss tensor (scalar), is already scaled by accumulated grads and possibly scaled by Mixed Precision.
```
scaled by accumulated grads? are you saying this is the accumulated loss or not?

Reply:
Reformulated.

Reply:
Done !